### PR TITLE
Feature: Collapsible

### PR DESCRIPTION
**Pull request by mrfrac**
### Description

PR to add new feature: collapsible.

PR includes:
- 2 sb stories
  - normal operating mode (no animation)
  - animation demo (angular animation capabilities)

PR **NOT** includes at this moment:
- tests
- not fully comply with the Radix Collapsible API (verfication required)

want to get feedback and critics!

### CODE REVIEW

File: packages/primitives/collapsible/src/collapsible-root.directive.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    private setPresence(): void {
+        if (!this.contentDirective) {
+            return;
+        }
+
+        this.contentDirective.elementRef.nativeElement.setAttribute('data-state', this.getState());
+
+        if (this.isOpen()) {
```
Comment (pimenovoleg) [inline packages/primitives/collapsible/src/collapsible-root.directive.ts:78]: You can try using `usePresence` here.

File: packages/primitives/collapsible/src/collapsible-root.directive.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    @Input() disabled = false;
+    @Input() set open(value: boolean) {
+        if (value !== this._open) {
+            this.openChange.emit(value);
+        }
+
+        this._open = value;
+    }
+    @Output() openChange = new EventEmitter<boolean>();
+    @ContentChild(RdxCollapsibleContentDirective) contentDirective?: RdxCollapsibleContentDirective;
```
Comment (pimenovoleg) [inline packages/primitives/collapsible/src/collapsible-root.directive.ts:43]: you can use `contentChild` with InjectToken from ContentDirective

Comment (mrfrac) [reply packages/primitives/collapsible/src/collapsible-root.directive.ts]: Done!

**Comment by mrfrac** [comment]
some additions:

- fix external triggering capability
- add SB story for external triggering feature
- add mdx docs

feel free to comment and feedback here. at this moment my work is finished. removing draft status for this PR

**Review by pimenovoleg — APPROVED** [review]

**Review by pimenovoleg — APPROVED** [review]

**Pull request merged**